sci-geosciences/josm-bin/josm-bin-18004.ebuild

```
@@ -0,0 +1,32 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit java-utils-2 desktop
+
+DESCRIPTION="Java-based editor for the OpenStreetMap project"
+HOMEPAGE="https://josm.openstreetmap.de/"
+SRC_URI="https://josm.openstreetmap.de/download/josm-snapshot-${PV}.jar"
+
+LICENSE="GPL-2"
```

One last thing, I think(?) this should be:
```suggestion
LICENSE="Apache-2.0 GPL-2+ GPL-3"
```
Haven't looked that deep into it but:
```
JOSM, and all its integral parts, are released under the GNU General 
Public License v2 or later.
```
The later is there.

And:
```
The JOSM repository contains files licensed under the Apache License, Version 2.0
(as stated at the beginning of the file). When compiled with any of this code
included, the whole software is licensed under GPL v3.
```